### PR TITLE
Feature/web 649

### PR DESCRIPTION
**Pull request by jasmeet17**
1. Update build script
2. Update content

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -15,7 +17,15 @@ end
 
 desc "Run the live updating development server"
 task :development_server do 
-  sh %{ EXECJS_RUNTIME=Node bundle exec middleman server } 
+	command = "EXECJS_RUNTIME=Node HOST=" + host + " bundle exec middleman server"
+  # sh %{ EXECJS_RUNTIME=Node bundle exec middleman server }
```
Comment (dlight-bean) [inline Rakefile]: shouldn't have commented out code

File: Rakefile [hunk 2]
```diff
@@ -3,6 +3,8 @@ require 'fileutils'
 
 CLOBBER.include('build')
 
+host =  ENV['HOST'] || 'www.beanstream.com'
```
Comment (dlight-bean) [inline Rakefile]: "host" is ambiguous. It should be something like swaggerHost

File: source/docs/forms/link_builder.html.erb [hunk 3]
```diff
@@ -2,7 +2,7 @@
 title: Link Builder
 layout: tutorial
 
-summary: This form will help you to create secure links to your Beanstream Hosted Payment Form.
+summary: This form will help you create secure links to your Hosted Payment Form.
```
Comment (dlight-bean) [inline source/docs/forms/link_builder.html.erb]: This should be "our Hosted Payment Form"

File: source/docs/forms/link_builder.html.erb [hunk 4]
```diff
@@ -16,7 +16,7 @@ breadcrumbs: false
 
 <h1>Hosted Payment Form - Link Builder</h1>
 
-<p>This form will help you to create secure links to your Beanstream Hosted Payment Form.
+<p>This form will help you create secure links to your Hosted Payment Form.
```
Comment (dlight-bean) [inline source/docs/forms/link_builder.html.erb]: This should be "our Hosted Payment Form"

File: source/docs/guides/apple_pay/index.md [hunk 5]
```diff
@@ -14,20 +14,20 @@ navigation:
 
 # Apple Pay
 
-**Apple with Beanstream is currently in a close Beta trial period. Contact your account manager if you would like early access.**
+**Apple with us is currently in a close Beta trial period. Contact your account manager if you would like early access.**
```
Comment (dlight-bean) [inline source/docs/guides/apple_pay/index.md]: Should be: 
Our Apple Pay is currently..

File: source/docs/sign_up/index.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ cards:
         icon: checkmark
         link: http://www.beanstream.com/pricing/
     -
-        title: Partner with Beanstream
+        title: Partner with us
```
Comment (dlight-bean) [inline source/docs/sign_up/index.md]: Should us be capitalized?

File: source/includes/na/getting_started/_create_account.md.erb [hunk 7]
```diff
@@ -73,7 +73,7 @@ Here's what you'll need, here's the time frame, here are the steps...
 <% end %>
 <div class='block-highlight hidden'><p></p></div>
 
-Request access to Onboarding APIs: http://developer.beanstream.com/documentation/onboarding-api/#sign_up_for_access (Note: this just sends email)
+Request access to Onboarding APIs: http://developer.bambora.com/documentation/onboarding-api/#sign_up_for_access (Note: this just sends email)
```
Comment (dlight-bean) [inline source/includes/na/getting_started/_create_account.md.erb]: should this be dev.na.bambora.com?

File: source/includes/na/getting_started/_create_account.md.erb [hunk 8]
```diff
@@ -119,7 +119,7 @@ Request access to Onboarding APIs: http://developer.beanstream.com/documentation
 
 <%= javascript_include_tag 'js/create_test_account.js' %>
 
-Request access to SproutPOS SDKs: http://developer.beanstream.com/documentation/sprout/ (Note: this just sends email)
+Request access to SproutPOS SDKs: http://developer.bambora.com/documentation/sprout/ (Note: this just sends email)
```
Comment (dlight-bean) [inline source/includes/na/getting_started/_create_account.md.erb]: should this be dev.na.bambora.com?

File: Rakefile [hunk 9]
```diff
@@ -3,6 +3,8 @@ require 'fileutils'
...
+
 desc "Build the static site"
 task :build_site do
   sh %{ bundle exec middleman build --verbose }
```
Comment (e-wipond) [inline Rakefile]: I think we need to pass in the HOST env variable here for the `:build_site` task as well. I think this is the one that Bamboo runs when deploying, so we need to make sure that works. Right now it's failing: https://bamboo.beanstream.com/browse/DR-DEV42

File: lib/custom_helpers.rb [hunk 10]
```diff
@@ -85,7 +85,7 @@ def get_all_swagger_specs(swagger_url, output_dir)
       end
     else
       # do it the old way
-      default_payments_spec = "1-0-2.json"
+      default_payments_spec = "1-0-3.json"
```
Comment (e-wipond) [inline lib/custom_helpers.rb]: @dlight-bean Is this code still necessary in this version of the dev portal, or will the endpoint be using the new format when this is live? 

If not, when it's falling back to this block of code, will version 1-0-3 be the one being downloaded?

File: source/docs/references/merchant_API/index.md [hunk 11]
```diff
@@ -20,9 +20,9 @@ You can either interact with the API directly or through a SDK (a library of hel
 
 Note: this doc is limited to the Payment, Reporting, Payment Profile, and Tokenization APIs. You may also be interested in the Onboarding API.
 
-## v1.0.2
+## v1.0.3
```
Comment (e-wipond) [inline source/docs/references/merchant_API/index.md]: Should we still link to previous versions (v1.0.2) here, the way the onboarding API does ([link](https://developer.beanstream.com/docs/references/onboarding_API/)). Also, do we have Change Logs for version 1.0.3?

File: source/includes/na/getting_started/_create_account.md.erb (hunk 7 above)
Comment (e-wipond) [inline source/includes/na/getting_started/_create_account.md.erb]: As far as I can tell, this partial file isn't actually part of the dev portal anywhere. It isn't isn't included by any other file. Should probably just delete it.

File: source/stylesheets/_site_primary_colors.scss [hunk 12]
```diff
@@ -2,13 +2,13 @@
...
-// $primary-light: rgb(202, 172, 223) ($faded-violet)
-// $primary-dark: rgb(69, 36, 89) ($amethyst)
+$primary: rgb(120, 71, 181); 			//($primary)
+$primary-light: rgb(202, 172, 223); 	//($faded-violet)
+$primary-dark: rgb(69, 36, 89); 		//($amethyst)
 
-$primary: #067aed;
-$primary-light: #71afed;
-$primary-dark: rgb(5, 110, 214);
+// $primary: #067aed;
```
Comment (e-wipond) [inline source/stylesheets/_site_primary_colors.scss]: Should probably just delete these lines. No need to keep reference to old Beanstream colors.

**Pull request merged**